tools/pysa_integration_tests/run.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import logging
+import os
+from pathlib import Path
+
+from utils import run_pysa_integration_test, LOG
```

Comment:
nitpick: You shouldn't import `LOG` from another module. `LOG` is intended to be redefined in each module so users can skip logging from a specific module.